### PR TITLE
Fixes issue #30

### PR DESCRIPTION
**Pull request by tmadruga**
- Add alt text, center alignment to images.
- Use math directive for equations.
- Cleaned up extra images in figure folder.
- Did not change too much of the content, but moved paragraphs around to make it more readable.
Note: Moved this chapter up in the toc to be after the refresher on equation of a line since it seemed to flow better.

### CODE REVIEW

**Comment by lindseyc** [comment]
@googlebot I consent.

File: ac1/_sources/regression_and_line_of_best_fit/interpreting_slope.rst [hunk 1]
```diff
@@ -8,78 +8,100 @@
...
 ==================
 
-When interpreting the relationship between two variables, one of the key
-concepts is interpreting slope. While the correlation coefficient tells you how
-strong the relationship between two variables is, the slope tells you the
-magnitude of that relationship. In other words, the correlation helps you answer
-the question “If variable A changes, will variable B change?”, while the slope
-helps you answer the question “If variable A changes, *how much* will variable B
-change?”.
+Now that you have reviewed the section on equation of a line, it's important to
```
Comment (raskutti) [inline ac1/_sources/regression_and_line_of_best_fit/interpreting_slope.rst]: Shorten this sentence

"Now that you have reviewed linear equations, this section outlines how that relates to data analysis." 
or something like that

File: ac1/_sources/regression_and_line_of_best_fit/interpreting_slope.rst [hunk 2]
```diff
@@ -8,78 +8,100 @@
...
+.. math::
+   \frac{rise}{run} = \frac{\text{change in y}}{\text{change in x}}
 
-.. image:: figures/SAT_Math_and_Earnings.png
 
+You've already learned about the correlation coefficient, *r*, which tells you
+how strong the relationship between two variables is. Slope tells you something
+different. The slope tells you the **magnitude** of that relationship. In other
+words, the correlation helps you answer the question “If variable A changes,
+*will* variable B change?”, while the slope helps you answer the question “If
```
Comment (raskutti) [inline ac1/_sources/regression_and_line_of_best_fit/interpreting_slope.rst]: Slightly change the wording

"If variable A changes, how confident are you that variable B will change?" or something like that

**Review by raskutti — APPROVED** [review]
Very nice work. Please remove pavement.py from this PR, and address the minor comments.

Also, some of the features (e.g. ..math) do not render in Github, so make sure these work in runestone. Once you see these all work, I trust you :)

**Comment by tmadruga** [comment]
@raskutti changes addressed.

**Review by raskutti — APPROVED** [review]

**Pull request merged**